templates/docker-compose.yml.j2

```diff
@@ -50,6 +50,8 @@ services:
       --max-connections={{ nim_waku_p2p_max_connections }}
       --store-capacity={{ nim_waku_store_capacity }}
       --dns4-domain-name={{ nim_waku_dns4_domain_name }}
+      --sqlite-store={{ nim_waku_sqlite_store }}
+      --sqlite-retention-time={{ nim_waku_sqlite_retention_time }}
```

Comment:
If store is disabled the `--sqlite-retention-time` flag should be conditional.

Also, are we sure the name of the flag should mention a specific database? Wouldn't it make more sense for the name of the flag to be db-agnostic? Like `--database-retention-time` or `--db-retention-time`?

Reply:
Good point re the naming. My understanding is that we want to consolidate (and significantly simplify!) store config when we define a general store interface which should be agnostic to the specific store implementation behind it (whether in-memory, any DB technology, or a hybrid). For now the sqlite store is a beta feature behind a very specific feature flag to allow us to test the concept. @kaiserd, any thoughts?

Reply:
Agree with @jm-clius . I would suggest making this part of the PR that introduces the new store interface.
Also, we plan to introduce subcommands soon, which allows making flags like this conditional.